### PR TITLE
Support datastore custom attribute as To: address

### PR DESCRIPTION
**Pull request by kremerpatrick**
Support for a custom attribute added to the datastore containing an additional To: address. Gives vCenter admins the ability to configure recipient directly in vCenter without having to modify and push a new JSON secret.  

Testing: Tested on vCenter 7 and VMC SDDC M9 with VEBA 0.3 release. Validated successful email delivery with all possible combinations of: authenticated SMTP, unauthenticated SMTP, custom attribute present, and custom attribute not present

Signed-off-by: Patrick Kremer <pkremer@vmware.com>

### CODE REVIEW

File: examples/powercli/datastore-usage-email/README.md [hunk 1]
```diff
@@ -16,7 +16,14 @@ git checkout master
 
 Step 2 - Update `stack.yml` and `vc-datastore-config.json` with your environment information
 
-<B>Note:</B> leave SMTP_USERNAME and SMTP_PASSWORD blank if you do not want to use authenticated SMTP
+**Note:**
```
Comment (embano1) [inline examples/powercli/datastore-usage-email/README.md:19]: Can you make these notes please in the form of

```
> **Note:** ...
```

**Comment by embano1** [comment]
@lamw can you please take a look?

**Comment by embano1** [comment]
@kremerpatrick can you please reword the title of the commit to something like

`Support datastore custom attribute as To: address`

Please also describe how you tested this function.

File: examples/powercli/datastore-usage-email/README.md (hunk 1 above)
Comment (kremerpatrick) [reply examples/powercli/datastore-usage-email/README.md:19]: Meaning include the front > which makes it render like this: 

image

Comment (kremerpatrick) [reply examples/powercli/datastore-usage-email/README.md]: Fixed

**Comment by embano1** [comment]
Thx @kremerpatrick for your quick response. Looks like you missed the newline in the commit message. Instead of `` it shows `Support datastore custom attribute as To: address. Tested on vCenter …`.

Please also update this PR with the test information.

**Comment by kremerpatrick** [comment]
Testing: Tested on vCenter 7 and VMC SDDC M9 with VEBA 0.3 release. Validated successful email delivery with all possible combinations of: authenticated SMTP, unauthenticated SMTP, custom attribute present, and custom attribute not present

**Comment by embano1** [comment]
@kremerpatrick this needs a rebase due to a change in the README from another PR.

**Comment by kremerpatrick** [comment]
Fixing the mess.

**Pull request closed without merging**